src/core/MenuItem/style.ts

```diff
@@ -25,33 +23,44 @@ export const StyledMenuItem = styled(MenuItem)`
     const primary = colors?.primary[400];
 
     return `
-      padding: ${spacings?.s}px ${spacings?.m}px;
+      padding: ${spacings?.s}px ${spacings?.m}px
 
       .primary-text {
-        font-weight: ${selected ? fontWeights?.semibold : fontWeights?.regular};
+        font-weight: ${selected ? fontWeights?.semibold : fontWeights?.regular}
       }
 
       &.MuiListItem-root.MuiListItem-button {
-        background-color: transparent;
+        background-color: transparent
 
         &:hover {
-          background-color: ${colors?.gray[100]};
+          background-color: ${colors?.gray[100]}
         }
       }
 
       &:hover {
+        background-color: ${colors?.gray[100]};
         svg {
-          color: ${selected ? primary : colors?.gray[500]};
+          color: ${selected ? primary : colors?.gray[500]}
+        }
+      }
+
+      &.Mui-selected {
+        background-color: white !important;
+        &:hover {
+          background-color: ${colors?.gray[100]} !important;
+        }
+        .primary-text {
+          font-weight: ${fontWeights?.semibold};
         }
       }
 
       &:active {
         svg {
-          color: ${primary};
+          color: ${primary}
```

Comment:
In general, please add semi colons to all the lines that don't have them

Reply:
Ahh, the `yarn lint` removes the semicolons from the code :(

Reply:
Fixed them all :)